storage/schemadb/src/lib.rs

```diff
@@ -216,6 +216,7 @@ impl DB {
         let mut db_opts = rocksdb::Options::default();
         db_opts.create_if_missing(true);
         db_opts.create_missing_column_families(true);
+        db_opts.set_max_open_files(1000);
```

Comment:
1024 for nice computer sciencey numbers? ;-)

Reply:
Also this should probably be configurable.

Reply:
I suspect there are performance considerations here? Though I don't we'd see anything in cluster test.

Reply:
Yes. I don't know whether I should land this or I make a refactor to create an option file that can be imported by our db.

Reply:
@davidiw theoretically the bigger this number, the better the performance. But I think 1k is good enough compared to inf.